### ISSUE TITLE
test TestAccMetalSSHKey_projectImportBasic failures

### ISSUE THREAD

**Issue by displague** [equinix/terraform-provider-metal#24]
From SSH Key projectImportBasic: 

```
--- FAIL: TestAccMetalSSHKey_projectImportBasic (2.27s)
    testing.go:569: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) {
        }
        
        
        (map[string]string) (len=1) {
         (string) (len=10) "project_id": (string) (len=36) "96e8fa35-58df-487b-be70-63d05f7a9ea7"
        }
        
=== RUN   TestAccMetalSSHKey_importBasic
```

_Originally posted by @t0mk in https://github.com/equinix/terraform-provider-metal/issues/21#issuecomment-771653885_

**Comment by displague** [comment]
This test is failing regularly in the PR where this issue was identified.

**Comment by displague** [comment]
This should be resolved with #44.
